### ISSUE TITLE
Can't sort the contact list

### ISSUE THREAD

**Issue by MikeKMiller** [mautic/mautic#13737]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.2.4

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What happened?

The table order remains unchanged when clicking on the headers of the contact list table, additionally changing number of rows changes nothing.

Previously opened as #13313 by @patrykgruszka , however was closed by @RCheesley without resolution.

### How can we reproduce this issue?

Step 1: Step 1: Add at least two contacts
Step 2: Click on the contact list table header
![302291549-ce806e7f-c157-4a46-8101-796b21d70c88](https://github.com/mautic/mautic/assets/15093231/0a376b8f-3b63-4325-878a-626b1d408977)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by PauperAnteFestum** [comment]
We must insist on this point because it is extremely important to sort the leads. It makes the leads table almost useless if it cannot be sorted.

**Comment by patrykgruszka** [comment]
This bug affects version 5.0.4, but works fine on the current development version. This issue is already addressed and will be fixed in the next release - 5.1.0.

**Comment by PauperAnteFestum** [comment]
> Este error afecta a la versión 5.0.4, pero funciona bien en la versión de desarrollo actual. Este problema ya se solucionó y se solucionará en la próxima versión: 5.1.0.

That's great to hear, dear! It would be quite a relief to have this feature available again. Do you have any estimated date on when this version will be available?

**Comment by pjrobertson** [comment]
I can confirm this is fixed. In the future, it would be really helpful for other users to list the commit that fixes a bug. That way, users can patch/pull that single commit to fix the issue without having to wait for a new release!

For those coming across this issue, the commit that fixed the problem is cc0027778a911ca22837d52944571ae571cbaadc

If you want to fix this now without waiting for 5.x, then you can run the following:

```bash
curl https://github.com/mautic/mautic/commit/cc0027778a911ca22837d52944571ae571cbaadc.patch | sudo patch -p1
bin/console mautic:a:g
```

**Comment by RCheesley** [comment]
As mentioned, this is already fixed so we don't need another issue. Thanks for posting the commit and instructions for how to apply it @pjrobertson.

For all your info, the best way to get a release out, is help to test the things that are blocking it. Testing can be done entirely in the browser, and we publish in Slack each week the things which we need help with.

Check here for more info: https://mau.tc/tester
